src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -2886,4 +2886,22 @@ public override void Accept(OperationVisitor visitor)
 
         protected override OperationKind ExpressionKind => OperationKind.None;
     }
+
+    /// <summary>
+    /// Providing implementation as OperationKind.None. This implementation is sufficient because the node doesn't survive initial binding.
+    /// </summary>
+    internal partial class DeconstructionLocalPendingInference
+    {
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitNoneOperation(this);
```

Comment:
If you could, it would be great.

---
In reply to: [68797293](https://github.com/dotnet/roslyn/pull/12224#discussion_r68797293) [](ancestors = 68797293)